gtm/variable_template.tpl

```diff
@@ -49,6 +49,13 @@ ___TEMPLATE_PARAMETERS___
       }
     ]
   },
+  {
+    "type": "CHECKBOX",
+    "name": "hasItems",
```

Comment:
Can you change this to "addRequestDataToItems" as I think that makes it clearer what it is doing? Thanks

Reply:
Thank you so much for your opinion. I agree with your opinion.
I changed this to “addRequestDataToItems”.